packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -378,9 +396,21 @@ const ReleaseDetailsBody = () => {
 
   if (isError || isReleaseError || !release) {
     return (
-      <ContentLayout>
-        <AnErrorOccurred />
-      </ContentLayout>
+      <Redirect
+        to={{
+          pathname: '/plugins/content-releases',
+          state: {
+            errors: [
+              {
+                code: releaseError?.code,
+              },
+              {
+                code: releaseActionsError?.code,
+              },
```

Comment:
I think you should only create the objects is the error variable is truthy. Because here it could be:

```js
errors: [
  {
    code: undefined,
  },
  {
    code: undefined,
  }
]
```